README.md

```diff
@@ -1 +1,189 @@
-# git
\ No newline at end of file
+<!-- 
+Headings
+Extended this comment over multi lines
+ -->
+
+# Sample .md file
+
+Below are sample snippets that can be used to quickly create/edit `.md` file.
+
+## Table of Contents
+* [Heading](#heading)
+* [Italics](#italics)
+* [Bold](#bold)
+* [Strikethrough](#strikethrough)
+* [BlockQuote](#blockquote)
+* [Links](#links)
+* [Ordered List](#ordered-list)
+* [Unordered List](#unordered-list)
+* [Inline Code Block](#inline-code-block)
+* [Code Blocks](#code-blocks)
+    * [bash](#bash)
+    * [python](#python)
+    * [javascript](#javascript) 
+* [Tables](#tables)
+* [Task lists](#task-lists)
+
+
+
+
+### Heading
+#### Syntax: 
+```
+#H1
+##H2
+###H3
+```
+#### Output:
+## H1
+### H2
+#### H3
+
+
+### Italics
+#### Syntax:
+```
+_ Text _
+```
+#### Output:
+_text_
+
+
+### Bold
+#### Syntax:
+```
+__ Text __
+```
+#### Output:
+__text__
+
+
+### Strikethrough
+#### Syntax:
+```
+~~ Text ~~
+```
+#### Output:
+~~Text~~
+
+
+### BlockQuote
+#### Syntax:
+```
+>Statement 1
+>Statement 2
+>Statement 3
+```
+#### Output:
+>Statement 1 \
+>Statement 2 \
+>Statement 3
+
+
+### Links
+#### Syntax:
+```
+[Display_Link_Name](link_pathname "hoverover_text")
```

Comment:
Please mention how we stack up links to bottom of document, to improve readability.